store/sqlstore/upgrade.go

```diff
@@ -19,7 +19,8 @@ import (
 )
 
 const (
-	CurrentSchemaVersion   = Version5360
+	CurrentSchemaVersion   = Version5370
```

Comment:
Did we recently change how we set versions? I thought we were setting this to the newest version but now I see we are in sync with the on-prem release. Of course the issue remains that whatever we set here is not 100% reflective of the schema state.

Reply:
This info probably escaped from me, I don't remember a recent decision change about versioning as well. Going to update the version to `5380`. Then we should also save the schema version in `upgradeDatabaseToVersion538` right?

Reply:
Yes, from what I recall that's what we used to do. But first let's see if @jwilander has any further input on this.

Reply:
There was no decision to change how we did it so we should set it to the newest version and save the schema version in `upgradeDatabaseToVersion538`